cocos/2d/framework/ui-renderer.ts

```diff
@@ -168,7 +168,6 @@ export class UIRenderer extends Renderer {
      */
     @type(Material)
     @displayOrder(0)
-    @displayName('CustomMaterial')
```

Comment:
Why remove it?

Reply:
handle with a custom approach to adding i18n

code: ./app/builtin/scene/source/script/export/dump/encode.ts

<img width="1117" alt="image" src="https://github.com/user-attachments/assets/d613e23b-02b3-490d-b221-c0df5941cde7">